torch/nn/parallel/distributed.py

```diff
@@ -290,15 +290,6 @@ class DistributedDataParallel(Module):
         Using ``DistributedDataParallel`` in conjunction with the
         :ref:`distributed-rpc-framework` is experimental and subject to change.
 
-    .. warning::
-        The ``gradient_as_bucket_view`` mode  does not yet work with Automatic
-        Mixed Precision (AMP). AMP maintains stashed gradients that are used for
-        unscaling gradients. With ``gradient_as_bucket_view=True``, these
-        stashed gradients will point to communication buckets in the first
-        iteration. In the next iteration, the communication buckets are mutated
```

Comment:
Could you clarify how this issue was fixed so that the stashed gradients are no longer mutated?

Reply:
the new amp scaler implementation in PyTorch does not use stashed gradients at all,  so it is not affected by how gradient is calculated